### DISCUSSION TITLE
Add "Login with different Account"

### DISCUSSION THREAD

**Discussion by appinteractive** [tinacms/tinacms#3864] [Help]
Currently, I prepare a site for a customer and need to change accounts for that, but the login screen forces me to log in with my current GitHub account, not the one used for the site.  

I can imagine others having similar issues in production environments. It's not uncommon to have multiple accounts, and on a lot of sites you get the ability to switch like on Google.

To solve this, please add an ability to switch accounts or login with another one.

<img width="544" alt="grafik" src="https://user-images.githubusercontent.com/432828/234559970-879abab3-f5eb-4a01-989b-68797e4da997.png">

**Comment by logan-anderson** [comment]
@appinteractive This is great feedback
